wrapper/src/test/java/integration/refactored/container/TestDriverProvider.java

```diff
@@ -92,6 +99,51 @@ public void beforeEach(ExtensionContext context) throws Exception {
                   // Enable all proxies
                   ProxyHelper.enableAllConnectivity();
                 }
+
+                if (TestEnvironment.getCurrent().getInfo().getRequest()
```

Comment:
This logic doesn't need for most of the test scenarios. Would you consider to relocate it to MakeSureFirstInstanceWriterExtension? It might also need to add this extension to some other test classes.

Reply:
Another option is to create another extension with this logic.